linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_suid_privilege_function/tests/other_key.pass.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# packages = audit
+
+echo '-a always,exit -F arch=b32 -S execve -C gid!=egid -F egid=0 -F key=my_setgid-audit-rule' > /etc/audit/rules.d/privileged.rules
+echo '-a always,exit -F arch=b64 -S execve -C gid!=egid -F egid=0 -k my_setgid-audit-rule' >> /etc/audit/rules.d/privileged.rules
+echo '-a always,exit -F arch=b32 -S execve -C uid!=euid -F euid=0 -k my_setuid-audit-rule' >> /etc/audit/rules.d/privileged.rules
+echo '-a always,exit -F arch=b64 -S execve -C uid!=euid -F euid=0 -F key=my_setuid-audit-rule' >> /etc/audit/rules.d/privileged.rules
```

Comment:
The OVAL is not checking the `key` value since it is an arbitrary string of text. So, this test scenario could also be removed.

Reply:
Okay I drop it.